Add roadmap and make FAQ more discoverable (#51)
agama-project/agama-project.github.io: docs/roadmap.md

---
sidebar_position: 4
---

# Roadmap

Agama is a project under constant development and priorities are re-evaluated every few weeks. This
page offers an overview on the current state of the ever-changing development plans.

## Agama 12 (latest released version)

> - New user interface based on PatternFly v6.

Reviewer: np: what about linking PatternFly?

Author: I will do on the corresponding blog post. I'm trying to keep this relatively raw (easier for updating via copy&paste).

> - Revamped "Storage" section at the UI. Support for LVM is temporarily removed.

Reviewer: ```suggestion
- Revamped "Storage" web user interface. Support for LVM is temporarily removed.
```

> - First version of a new "Authentication" section at the UI replacing the previous "Users" one.

Reviewer: web user interface

---
sidebar_position: 4
---

# Roadmap

> Agama is a project under constant development and priorities are re-evaluated every few weeks. This

Reviewer 2: It's actually every few days :D But weeks sound like we know what we are doing ;)
